RELEASE-LEADS.md

```diff
@@ -119,104 +109,62 @@ Make a copy of the
 empty it out and send it to the WG leads of the respective project (serving or
 eventing) to fill in. Coordinate with both serving and eventing leads.
```

Comment:
For the release notes, can we incorporate in our workflow the tooling as used here: https://github.com/knative-sandbox/eventing-kafka-broker/pull/250 ?

Reply:
I will do you one better, drop that script and use this: https://github.com/knative-sandbox/.github/pull/69

Reply:
That's cool!